contracts/LifToken.sol

```diff
@@ -455,47 +456,64 @@ contract LifToken is Ownable, ERC20, SafeMath, PullPayment {
 
     }
 
-    // ERC20 transfer method but with data parameter.
+    //ERC20 token approve method with data call/log option.
+    function approveData(address spender, uint value, bytes data, bool doCall) onStatus(3,4) returns (bool success) {
```

Comment:
This is defined on the ERC20 token standard, its one of the few things that we shouldnt change to follow the standard, if we were on another function we can. And the spender not necessarily needs to eb a token holder, it can be an address without any tokens too :).